MovingBlocks/Terasology
# Critical fix of gradle build - add protobuf plugin

### Contains

Fixes of issue #2873
Add protobuf gradle plugin, update version of protoc, update proto files, remove useless tasks, remove redundant files.

### How to test

Change something in *.proto files. 
And then recompile module: `$ ./gradlew engine:recompileProto`
See that related java files were updated.
